schemas/task.yml

```diff
@@ -187,7 +193,6 @@ required:
   - provisionerId
   - workerType
   - schedulerId
-  - taskGroupId
```

Comment:
These changes are bad... because we don't get any validation that output is correct.
Suddenly we won't get an error serverside if we return the wrong thing and clients will have to handle the case where `taskGroupId` is missing.

Also `priority` isn't required in the createTask call... There is a big difference in what is required.